danger-klaxit/lib/danger_plugin.rb

```diff
@@ -66,22 +66,25 @@ def warn_for_public_methods_without_specs
     return failure("`spec` directory is missing") unless Dir.exist?("spec")
 
     spec_regex = /describe "([#.][^"]+)"/
-
-    new_public_methods_by_ruby_file.each do |file, method_details|
-      file = new_name_for_file(file)
-      spec_file = "spec/" + file.sub(".rb", "_spec.rb").sub("app/", "")
-      next warn("No spec found for file #{file}.") unless File.exist?(spec_file)
-
-      specs = IO.foreach(spec_file)
-                .map { |line| line[spec_regex, 1] }
-                .to_set
-                .tap { |set| set.delete(nil) }
-      method_details.each do |details|
-        next if specs.include?(details.name_with_prefix)
-
-        warn("Missing spec for `#{details}`", file: file, line: details.line)
+    db_files_regex = %r(db\/.*)
```

Comment:
Do you wear both a belt and suspenders?

![image](https://user-images.githubusercontent.com/11378424/95225019-9f8dfb00-07fb-11eb-867b-d0e6d674d87b.png)

`%r(db/.*)` or `/db\//` would both do the trick here!

Moreover, I think you could just use `String#start_with?` which is more efficient, more correct (it is the beginning of the path) and straight to the point!